src/hatti/ona/forms.cljs

```diff
@@ -193,6 +198,19 @@
                 (nil? thumb)          answer
                 :else                 [:a {:href image :target "_blank"}
                                        [:img {:width "80px" :src thumb}]]))
+       :vid (let [image (:download_url answer)
+                  thumb (or (:small_download_url answer) image)
+                  fname (last-url-param (:filename answer))]
+              (cond
+                (or (nil? answer)
+                    (string? answer)) answer
+                compact?              (format
```

Comment:
Not sure I understand, does this not compile sablono? Like if you use `[:a ...]` it doesn't work?